pkgs/top-level/python-packages.nix

```diff
@@ -26537,6 +26537,30 @@ EOF
     };
   };
 
+  yowsup = buildPythonPackage rec {
```

Comment:
Please read the header of this file. We want to move new or updated expressions to `pkgs/development/python-modules/<name>/default.nix`